whipper/test/test_common_config.py

```diff
@@ -66,3 +66,25 @@ def testDefeatsCache(self):
         defeats = self._config.getDefeatsCache(
             'PLEXTOR ', 'DVDR   PX-L890SA', '1.05')
         self.assertEquals(defeats, True)
+
+    def test_get_musicbrainz_server(self):
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          'musicbrainz.org')
+
+        self._config._parser.add_section('musicbrainz')
+
+        self._config._parser.set('musicbrainz', 'server',
+                                 '192.168.2.141:5000')
+        self._config.write()
+
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          '192.168.2.141:5000')
+
+        self._config._parser.set('musicbrainz', 'server',
+                                 '192.168.2.141:5000/hello/world')
+        self._config.write()
+
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          '192.168.2.141:5000')
+
+        self._config._parser.remove_section('musicbrainz')
```

Comment:
This seems like something that should be done in a "tear down" function/method. What happens if the test breaks before it gets to this call?

Reply:
I agree, but I can't see a good way to handle it, since it's not common to the all (or even the majority) of the tests.